### DISCUSSION TITLE
Dead links - latest ImageMagick for Windows

### DISCUSSION THREAD

**Discussion by FrontierDK** [ImageMagick/ImageMagick#6013] [General]
Hi all.

With the latest / newest version, a lot of the links for Windows are dead links.

Can anyone upload the missing files?

**Comment by snibgo** [comment]
Please link to a page with dead links, and paste one of the links that is dead.

**Reply by FrontierDK** [reply]
The page is https://imagemagick.org/script/download.php and the following links are dead:

```
https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-60-portable-Q16-x86.zip
https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-60-portable-Q8-x64.zip
https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-60-portable-Q16-HDRI-x86.zip

```

**Comment by snibgo** [comment]
Thanks. Of those links, the first and third currently work, but https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-60-portable-Q8-x64.zip is broken.

I suspect a problem in the automated build and upload process, which will probably be fixed within a day or two, but a developer may comment.
